src/pudl/metadata/resources/eia.py

```diff
@@ -1211,12 +1213,164 @@
         "sources": ["eia860"],
         "etl_group": "static_eia",
     },
-}
+} | {
+    # out_eia__yearly_generators and out_eia__monthly_generators are defined here
+    f"out_eia__{freq}_generators": {
```

Comment:
* Migrated this from `mcoe.py` since this location is more intuitive based on the table names.
* Added this comment so that if one is grepping for table names, you'll find this location.